pkg/azurefile/controllerserver.go

```diff
@@ -584,6 +607,7 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 					return nil, status.Errorf(codes.Internal, "failed to GetStorageAccesskey on account(%s) rg(%s), error: %v", accountOptions.Name, accountOptions.ResourceGroup, err)
 				}
 			}
+			klog.V(2).Infof("SetAzureCredentials accountName(%s), accountKey(%s), secretName(%s), secretNamespace(%s)", accountName, accountKey, secretName, secretNamespace)
```

Comment:
don't print accountKey anywhere, otherwise that's a security issue!

Reply:
done